README.md

```diff
@@ -7,6 +7,15 @@ Static Resolver Bundle is designed to encapsulate the usage of static calls with
 - Provides a proxy factory to reduce complexity, enhance testability, simplify code maintenance, and enforce better coding standards.
 - Promotes a cleaner, more consistent and well-organized codebase.
 
+:::caution
+
+This bundle is public but will be treated like an internal bundle.
+It mostly wraps static calls to be able to inject static calls as services.
+This bundle is not intended to be used by third-party developers.
```

Comment:
Why not? It would be great to be able to *not* use all these static calls.

Reply:
https://github.com/pimcore/static-resolver-bundle/issues/70#issuecomment-2486230968
I would invite you to add to this discussion :)